portal-backend/depmap/dataset/models.py

```diff
@@ -105,7 +105,7 @@ class Dataset(Model):
     __mapper_args__ = {"polymorphic_identity": "dataset", "polymorphic_on": type}
 
     @property
-    def nominal_range(self):
+    def nominal_range(self): # No longer used
```

Comment:
I suggested that we get rid of this, but I think a cheap way we could add this back (one might say, the "right" way it should have been modeled actually) is determine the nominal range based on the *units* of the dataset.

Basically, all "gene effect" scores should be in the range something like (-1.1, 0.1), and all compound viability should be something like (0, 1) etc.

We'd probably have a little headache from how our units are written may not be consistent on different datasets, but I think determining the range from units makes more sense then needing to set it per-dataset.

Regardless, nothing to do here for now. Just mentioning it as a fyi